<launch>

	<!-- Record AR Tag corners -->
	<include file="$(find probability_filters)/launch/ar_tag_recorder.launch" />

	<!-- Record filter corners based on saved AR Tag position --> 
	<include file="$(find probability_filters)/launch/bounding_box_object_filter.launch" >
		 <arg name="only_record" value="True" />
	</include>


</launch>